ingestion/src/metadata/ingestion/source/metadata/atlas/client.py

```diff
@@ -42,7 +42,9 @@ def __init__(self, config: AtlasConnection, raw_data: bool = False):
         self._use_raw_data = raw_data
 
     def list_entities(self, entity_type="Table") -> List[str]:
-        response = self.client.get(f"/atlas/entities?type={entity_type}")
+        response = self.client.get(
+            f"/atlas/entities?type={self.config.entity_type if self.config.entity_type else entity_type}"
```

Comment:
@NiharDoshi99 will this condition be always true since the `entity_type` is a required field?

Reply:
Oh yes , let me remove this if condition 👍